### PR TITLE
Improved Readme

### PR DESCRIPTION
**Pull request by Aayush-hub**
## What is the change?
Improved Readme according to changes requested

## Related issue?
issue: #4 

## How was it tested?
Locally by contributor.

## Checklist:
Before you create this PR, confirm all the requirements listed below by checking the checkboxes `[x]`:

- [x] Have you followed the [Contribution Guidelines](https://github.com/ALPHAVIO/BlogSite/blob/master/CONTRIBUTING.md) while contributing.
- [x] Have you checked there aren't other open [Pull Requests](https://github.com/ALPHAVIO/BlogSite/pulls) for the same update/change?
- [x] Your submission doesn't break any existing feature.
- [x] Have you tested the code before submission?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -98,6 +109,37 @@ show collections
...
+
+
+
+- Mexili is a Free and Open Source organization whose motivation is to funnel learning and implementation through Open Source Software Development.
+
+
+
+
+
+
```
Comment (Pranav016) [inline README.md:140]: So many extra lines throughout the file

**Review by Pranav016 — CHANGES_REQUESTED** [review]
Good work, just some extra lines that were inserted need to be removed. Just give one line between any heading or sections. One more commit and it is ready to merge

**Comment by Aayush-hub** [comment]
@Pranav016 Done with the changes:)

**Comment by Pranav016** [comment]
@all-contributors please add @Aayush-hub for doc

**Pull request merged**